rust_snuba/src/factory.rs

```diff
@@ -79,6 +81,13 @@ impl ConsumerStrategyFactory {
 }
 
 impl ProcessingStrategyFactory<KafkaPayload> for ConsumerStrategyFactory {
+    fn update_partitions(&self, partitions: &HashMap<Partition, u64>) {
+        match partitions.keys().map(|partition| partition.index).min() {
+            Some(min) => set_global_tag("min_partition".to_owned(), min.to_string()),
+            None => set_global_tag("min_partition".to_owned(), "none".to_owned()),
+        }
```

Comment:
This could be written as
```suggestion
        let min = partitions
            .keys()
            .map(|partition| partition.index)
            .min()
            .map(|i| i.to_string())
            .unwrap_or("none".to_string());
        set_global_tag("min_partition".to_owned(), min);
```
but it's really a matter of opinion which is clearer.

Reply:
yeah i had that initially but thought the vertical drift isn't nice